maro/simulator/scenarios/cim/business_engine.py

```diff
@@ -109,6 +105,18 @@ def snapshots(self) -> SnapshotList:
         """SnapshotList: Snapshot list of current frame."""
         return self._snapshots
 
+    @property
+    def data_container(self) -> CimDataContainerWrapper:
```

Comment:
Could it be public?

Reply:
I added this property function only for testing use. I personally think that keeping such an interface is not a big problem, because users can always obtain this object by forcibly accessing internal properties. Leaving this interface can make the test code more elegant. What's your opinion?

Reply:
👌